pkg/reconciler/pipelinerun/resources/pipelinerunresolution_test.go

```diff
@@ -972,32 +1007,38 @@ func TestIsSkipped(t *testing.T) {
 		},
 	}, {
 		name:  "run-started",
-		state: oneRunStartedState,
+		tasks: oneRunStartedState,
 		expected: map[string]bool{
 			"mytask13": false,
 		},
 	}, {
 		name:  "run-cancelled",
-		state: runCancelled,
+		tasks: runCancelled,
 		expected: map[string]bool{
 			"mytask13": false,
 		},
```

Comment:
similar to above comments about combining cases together, i wonder if it would make sense to add a few new ones here too?

Reply:
will separate them, waiting for this to get in https://github.com/tektoncd/pipeline/pull/3744

Reply:
added new ones instead of combining